RecoCTPPS/Configuration/test/raw_data_test.py

```diff
@@ -0,0 +1,42 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("TotemStandaloneRawDataTest")
+
+# minimum of logs
+process.MessageLogger = cms.Service("MessageLogger",
+    statistics = cms.untracked.vstring(),
+    destinations = cms.untracked.vstring('cout'),
+    cout = cms.untracked.PSet(
+        threshold = cms.untracked.string('WARNING')
+    )
+)
+
+# raw data source
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring('file:/afs/cern.ch/user/j/jkaspar/public/run273062_ls0001-2_stream.root')
```

Comment:
If I use something like
```
/store/express/Run2016B/ExpressPhysics/FEVT/Express-v2/000/274/421/00000/
```
is it guaranteed that the config will stay working in few weeks, months? If not, what would be a safe "path" to our raw data? Thanks!